docs/src/user-guide/command-line-interface.md

```diff
@@ -186,35 +139,67 @@ npx eslint --global require --global exports:true
 
 #### `--parser`
 
-This option allows you to specify a parser to be used by ESLint. By default, `espree` is used.
+This option allows you to specify a parser to be used by ESLint.
+
+* **Argument Type**: String. Parser to be used by ESLint.
+* **Multiple Arguments**: No
+* **Default Value**: `espree`
+
+##### `--parser` example
+
+```shell
+# Use TypeScript ESLint parser
+npx eslint --parser @typescript-eslint/parser files.ts
+```
 
 #### `--parser-options`
 
-This option allows you to specify parser options to be used by ESLint. Note that the available parser options are determined by the parser being used.
+This option allows you to specify parser options to be used by ESLint. The available parser options are determined by the parser being used.
+
+* **Argument Type**: Key/Value pair. Separated by colon (`:`).
+* **Multiple Arguments**: Yes
 
-Examples:
+##### `--parser-options` example
 
 ```shell
-echo '3 ** 4' | npx eslint --stdin --parser-options=ecmaVersion:6 # fails with a parsing error
-echo '3 ** 4' | npx eslint --stdin --parser-options=ecmaVersion:7 # succeeds, yay!
+echo '3 ** 4' | npx eslint --stdin --parser-options ecmaVersion:6 # fails with a parsing error
+echo '3 ** 4' | npx eslint --stdin --parser-options ecmaVersion:7 # succeeds, yay!
 ```
 
 #### `--resolve-plugins-relative-to`
 
-Changes the folder where plugins are resolved from. By default, plugins are resolved from the current working directory. This option should be used when plugins were installed by someone other than the end user. It should be set to the project directory of the project that has a dependency on the necessary plugins. For example:
+Changes the folder where plugins are resolved from.
+
+* **Argument Type**: File path.
+* **Multiple Arguments**: No
+* **Default Value**: By default, plugins are resolved from the current working directory.
+
+This option should be used when plugins were installed by someone other than the end user. It should be set to the project directory of the project that has a dependency on the necessary plugins. If you use npm to install plugins, the path ends with `/node_modules/`.
+
+For example:
 
 * When using a config file that is located outside of the current project (with the `--config` flag), if the config uses plugins which are installed locally to itself, `--resolve-plugins-relative-to` should be set to the directory containing the config file.
 * If an integration has dependencies on ESLint and a set of plugins, and the tool invokes ESLint on behalf of the user with a preset configuration, the tool should set `--resolve-plugins-relative-to` to the top-level directory of the tool.
 
-### Specifying rules and plugins
+##### `--resolve-plugins-relative-to` example
+
+```sh
```

Comment:
```suggestion
```shell
```

This should fix the site build error.

```
9:34:06 PM: [11ty] 1. Having trouble rendering njk template ./src/user-guide/command-line-interface.md (via TemplateContentRenderError)
9:34:06 PM: [11ty] 2. "sh" is not a valid Prism.js language for eleventy-plugin-syntaxhighlight (via Error)
```